audio/fdk-aac/fdk-aac.SlackBuild

```diff
@@ -90,11 +92,12 @@ rm -f $PKG/{,usr/}lib${LIBDIRSUFFIX}/*.la
 find $PKG -print0 | xargs -0 file | grep -e "executable" -e "shared object" | grep ELF \
   | cut -f 1 -d : | xargs strip --strip-unneeded 2> /dev/null || true
 
-mkdir -p $PKG/usr/doc/$PRGNAM-$VERSION
-cat $CWD/$PRGNAM.SlackBuild > $PKG/usr/doc/$PRGNAM-$VERSION/$PRGNAM.SlackBuild
+mkdir -p $PKG/{install,usr/doc/$PRGNAM-$VERSION}
 
-mkdir -p $PKG/install
-cat $CWD/slack-desc > $PKG/install/slack-desc
+find . $CWD -type f \( -name "NOTICE" \) -exec cp {} $PKG/usr/doc/$PRGNAM-$VERSION/ \; -o \
```

Comment:
Also, it's easier to use cp directly instead of using find when you can see the files from the tarball

Reply:
Yeah, i'm just trying to be consistent across my slackbuilds.

Reply:
I do appreciate the tips though. I am still getting the hang of git/github and contributing to the community :)

Reply:
let's stay with the templates
it's trying to keep other people easy to read